lib/internal/errors.js

```diff
@@ -1375,3 +1375,4 @@ E('ERR_WORKER_UNSUPPORTED_EXTENSION',
 E('ERR_WORKER_UNSUPPORTED_OPERATION',
   '%s is not supported in workers', TypeError);
 E('ERR_ZLIB_INITIALIZATION_FAILED', 'Initialization failed', Error);
+E('ERR_WHATEVER_NAME', '%s : %s', Error);
```

Comment:
`ERR_INVALID_JSON_MODULE` or similar would probably be more appropriate.

An entry also needs to be added to `doc/api/errors.md`

Reply:
Thanks! Appreciated